gslib/tests/test_publicaccessprevention.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for publicaccessprevention command."""
+
+from __future__ import absolute_import
+
+from gslib.cs_api_map import ApiSelector
+import gslib.tests.testcase as testcase
+from gslib.tests.util import ObjectToURI as suri
+from gslib.tests.util import unittest
+
+
+class TestPublicAccessPrevention(testcase.GsUtilIntegrationTestCase):
+  """Integration tests for publicaccessprevention command."""
+
+  _set_publicaccessprevention_cmd = ['publicaccessprevention', 'set']
+  _get_publicaccessprevention_cmd = ['publicaccessprevention', 'get']
+
+  def test_off_on_default_buckets(self):
+    if self.test_api == ApiSelector.XML:
```

Comment:
Here, and for all the tests, you can instead use the SkipForXML decorator. You can have the decorator for the entire class as well. Checkout https://github.com/GoogleCloudPlatform/gsutil/blob/ee2cf7aefef96dcf1d0afd473723cff9fa823418/gslib/tests/test_context_config.py#L159